data/updates.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
This file will conflict with script generated one https://github.com/Code-for-All/lockdown/blob/2fe97699157b78d330cf373dc33102655ec880d5/data/updates.json

Reply:
How will this conflict? It looks similar? Is it just the fact that a `link` is not always available?

Reply:
I meant this PR will conflict because my PR also contains `updates.json`

Reply:
Ah yes, you're right. No worries, I'll fix that on my end, mine was a placeholder until yours was done 🙂